.luacheckrc

```diff
@@ -0,0 +1,15 @@
+globals = {"Test",
+  "config",
```

Comment:
@LuxoftAKutsan 
Common way to use "config" variable is use a global variable
Example:  
```
config.deviceMAC = "<MAC_value>"
```